db/DB.js

```diff
@@ -88,18 +88,25 @@ const DB = {
     const sha1 = getSha1(email);
 
     return await this._getSha1EntryAndDo(sha1, async aEntry => {
-      const removedSubscriber = await knex("subscribers")
-        .update({ email: null })
+      await knex("subscribers")
         .where("id", "=", aEntry.id)
-        .returning("*");
-      return removedSubscriber[0];
+        .del();
+      console.log("Removed subscriber ID: ", aEntry.id);
+      return aEntry;
+    }, async () => {
+      console.warn("removeSubscriber called with email not found in database.");
```

Comment:
Definitely not the email address - I want to avoid logging any plaintext emails as much as possible. And if we don't have a `subscribers` record, then we won't have `aEntry` or `aEntry.id`.

We will have the SES message ID in our logs, which should be enough to debug if we need to dig deep into any particular SES notification message.